### ISSUE TITLE
Does Not Show Graphichs files  in Win XP

### ISSUE THREAD

**Issue by AntonZab** [sirjuddington/SLADE#783]
**SLADE Version**: (3.1.1.5)
**Operating System**: (Win XP SP3)

**Issue Details**:
(Does Not Show Graphichs files  in Win XP in any wads)
https://drive.google.com/file/d/0B0oEDro3cJx-bnppc1Q5WnZOaFE/view?usp=sharing

**Comment by Gaerzi** [comment]
Please attempt to see a graphic file, open the console, and copy-paste the console log here so we can see what error messages it got.

**Comment by AntonZab** [comment]
00:11:23: SLADE - It's a Doom Editor
00:11:23: Version 3.1.1.5
00:11:23: Written by Simon Judd, 2008-2017
00:11:23: Compiled with wxWidgets 3.1.0 and SFML 2.4
00:11:23: --------------------------------
00:11:23: Windows Version: 5.1
00:11:23: Loading configuration
00:11:23: Loading resources
00:11:23: Loading icons
00:11:23: Loading entry types
00:11:23: Loading text languages
00:11:23: Loading text style sets
00:11:23: Loading colour configuration
00:11:34: Loading base resource
00:11:34: Base resource loaded
00:11:34: SLADE Initialisation OK
00:11:34: Read game configuration "hexen" + "zdoom"
00:11:34: Checking for updates...
00:11:35: Latest stable release: v3115 "3.1.1.5"
00:11:35: Latest beta release: v3120_b4 "3.1.2 Beta 4"
00:11:35: Already up-to-date
00:11:44: Opening archive G:/Games/doom2.wad
00:11:46: Warning: No fluidsynth soundfont set, MIDI playback will not work
00:11:46: Opening took 1581 ms

**Comment by AntonZab** [comment]
In all windows where images should be displayed - they are not. Instead of pictures glitches like in Zdoom where they forgot to put the picture on the wall

**Comment by Gaerzi** [comment]
What does the console show _after_ you have clicked on an image? Normally it should show  some message about initializing OpenGL the first time you trigger the creation of a display canvas (the area where the image is displayed, in simpler terms), so if it fails at this step there'll hopefully be an error message about it.

**Comment by sirjuddington** [comment]
Are your video drivers up to date?

**Comment by AntonZab** [comment]
Yes

**Comment by sirjuddington** [comment]
Closing this since winxp support will be discontinued soon
